app/src/screens/bank.js

```diff
@@ -6,12 +6,49 @@ import { Container, Section } from '../ui/layout';
 import { Text } from '../ui/typography';
 import NavBar from '../ui/nav-bar';
 import Card from '../ui/card';
+import Toast from '../ui/toaster/toast';
+
+function GameHistory() {
+  let { notice, history } = useGame();
+
+  if (!notice || !history.length) { return null; }
+
+  return (
+    <Section data-test-history>
+      <Text data-test-history-heading>History</Text>
```

Comment:
The data attribute kinda makes the text content get lost. Can `History` go on it's own line? The data attr is fine as is though since it's the only prop.